app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -61,5 +62,16 @@ class AuthRepositoryTest : ShouldSpec(
 
 			verify { firebaseAuth.currentUser!!.delete() }
 		}
+
+		should("return Profile data WHEN userProfile called") {
+			val profile = Profile(null, "-", "-")
+			every { firebaseAuth.currentUser!! } returns mockk {
+				every { photoUrl } returns null
+				every { displayName } returns "-"
```

Comment:
It is better to return some real values from mocks – this way it is easier to later check how the data could look.

Reply:
Done 😄